src/plugins/telemetry/schema/oss_plugins.json

```diff
@@ -4384,9 +4384,6 @@
         },
         "apm:enableServiceOverview": {
           "type": "boolean"
-        },
-        "apm:enableCorrelations": {
```

Comment:
FYI, You'd need to open a corresponding mapping update request in the [infra repo](https://github.com/elastic/infra/issues/new/choose) to have this key removed from the telemetry mappings.
If you don't care that it's not there in versions from 7.12, then you won't need to open an issue for that.

Reply:
We'll probably remove it from the mapping in the infra repo in 7.13. That way we'll be able to tell if users are still using it as a custom UI Setting for version 7.12.